suppy/simulator.py

```diff
@@ -302,13 +60,16 @@ def run(
         start_or_end_period: int,
         /,
         end_period: Optional[int] = None,
+        loops: Optional[int] = 1,
```

Comment:
`Optional[int]` means loops can be None

```suggestion
        loops: int = 1,
```

Reply:
The intended behavior is that when loops is not provided, it defaults to 1. Does the suggested change implement this behavior?